metaflow/plugins/argo/argo_workflows.py

```diff
@@ -488,6 +497,9 @@ def _process_parameters(self):
                 description=param.kwargs.get("help"),
                 is_required=is_required,
             )
+
+            if extra_attrs:
+                parameters[param.name]["extra_attrs"] = extra_attrs
```

Comment:
any reason to add the extra_attrs dict and not add the values directly to parameters[param.name]

Reply:
hmm, not really..

just collecting all of extra stuff at one place..
that's it..

let me know if this needs to change..

Reply:
Yeah it will be good to follow the same convention that we currently have for other attributes of parameters

Reply:
fixed